gammapy/utils/fitting/parameter.py

```diff
@@ -173,7 +173,7 @@ def autoscale(self, method="scale10"):
             value = self.value
             if value != 0:
                 power = int(np.log10(np.absolute(value)))
-                scale = 10 ** power
+                scale = float(10 ** power)
```

Comment:
Note, you could also use `numpy.float_power` there. Or even more simply `(10.0 ** power)`.

Reply:
Thanks for the review. I put `10. ** power` in  b41621f8.